generate_yaml/entrypoint.sh

```diff
@@ -75,11 +75,15 @@ main(){
     google_credentials=$INPUT_GOOGLE_CREDENTIALS
     launchdarkly_sdkkey=$INPUT_LAUNCHDARKLY_SDKKEY
     papertrail_url=$INPUT_PAPERTRAIL_URL
+    post_action_events_pub_sub_project_id=$INPUT_POST_ACTION_EVENTS_PUBSUB_PROJECT_ID
+    post_action_events_pub_sub_credentials_json=$INPUT_POST_ACTION_EVENTS_PUBSUB_CREDENTIALS_JSON
```

Comment:
These are project specific variables which shouldn't be in an action that is supposed to be generic.

What's the reason to put the variables here?

Reply:
If I understood correctly, this file is the responsible of creating the YAML file on the fly, containing the environment variables that are used in our projects after deploying to staging or production.

I have used the same approach as with other github secrets (i.e. google pubsub credentials used already in the app). Is there anything you think could be done differently?